src/Raven.Server/Routing/RequestRouter.cs

```diff
@@ -51,7 +51,7 @@ public RouteInformation GetRoute(string method, string path, out RouteMatch matc
             return tryMatch.Value;
         }
 
-        public async ValueTask<string> HandlePath(HttpContext context, string method, string path)
+        public async Task HandlePath(HttpContext context, string method, string path, Reference<string> database)
```

Comment:
Why did you make this change? This is `ValueTask` for a reason.

Reply:
We don't need the `database`, pass the `RequestHandlerContext` into this method and set the value as appropriate. That would save an allocation

Reply:
done